integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -375,11 +376,11 @@ private static void createDomainCrAndVerify(String adminSecretName,
     final String volumeName = "weblogic-domain-storage-volume";
     final String logHomeRootPath = "/scratch";
     // create the domain CR
-
+    String imagePoolPolicy = OKE_CLUSTER ? "Always" : "IfNotPresent";
```

Comment:
```suggestion
    String imagePoolPolicy = OKE_CLUSTER ? V1Container.ImagePullPolicyEnum.ALWAYS : V1Container.ImagePullPolicyEnum.IFNOTPRESENT;
```

Reply:
fluent spec takes string , not enum type